lib/ansible/plugins/connection/win_vmware_tools.py

```diff
@@ -0,0 +1,118 @@
+"""Docstring for public module."""
+from __future__ import absolute_import, division, print_function
+
+from pyVmomi import vim
+
+from ansible.plugins.connection.vmware_tools import Connection as VMwareToolsConnection
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    author: Deric Crago <deric.crago@gmail.com>
+    connection: vmware_tools
+    short_description: Execute modules via VMware Tools.
+    description:
+      - Execute modules via VMware Tools.
```

Comment:
Can we rename this as `vmware_win_tools` or `vmware_windows_conn` or something similar ?

Reply:
I'm not completely opposed to a rename, but I went with the `win_` prefix to follow the precedent set by the [Windows modules](https://docs.ansible.com/ansible/latest/modules/list_of_windows_modules.html).